libraries/ESP8266HTTPClient/src/ESP8266HTTPClient.h

```diff
@@ -135,8 +135,10 @@ class HTTPClient
 
     bool begin(String url);
     bool begin(String url, String httpsFingerprint);
+    bool begin(String url, const uint8_t httpsFingerprint[20]);
```

Comment:
Would be a good idea to add comments here saying which methods instantiate an axtls and which a bearssl object.

Reply:
Will do!